### ISSUE TITLE
Version 5.15.2 does not work anymore

### ISSUE THREAD

**Issue by guitar9** [nborrmann/jodel_api#84]
I updated the key but i get still error 477.

```
    api_url = "https://api.go-tellm.com/api{}"
    client_id = '81e8a76e-1e02-4d17-9ba0-8a7020261b26'
    secret_legacy = 'hyTBJcvtpDLSgGUWjybbYUNKSSoVvMcfdjtjiQvf'.encode('ascii')
    version_legacy = '4.47.0'
    secret = 'EUJTncIwqWKzUiqFzbzxfhneutfkUgcFENCIcceH'.encode('ascii')
    version = '5.15.2'
```

Does it work for you?

**Comment by statist32** [comment]
I have the same problem with this key

**Comment by Daedra22** [comment]
Same for me, even with the correct key for 5.16.1 it generates a request error, did they added something to the verification process? @nborrmann

**Comment by themultiplexer** [comment]
Yes they must have changed something :/ I hope sombody has a clue, because I have no experience how the api currently works. Sadly I have no time for debugging (or how to mitm jodel).

Sadly Christian Fibich has no time for implementing the latest authentication version ([issue](https://bitbucket.org/cfib90/ojoc/issues/39/registration-fails))

@nborrmann Do you have an idea how the latest signing mechanism works?

**Comment by Daedra223** [comment]
So whats the problem now @nborrmann are you too lazy? Do you have not enough time to fix the new version? Is it not possible?
Do you want money as a motivation then post a paypal link.

But dont ignore the questions explain the problem and whether its not possible to be solved, but do not do nothing ffs...

**Comment by SimonKrill** [comment]
Is there any way to get the API working? 
I am failing by creating a new account. Is this depending on the signing key?
